nixpkgs_review/nix.py

```diff
@@ -305,6 +313,8 @@ def nix_build(
         else "--no-allow-import-from-derivation",
     ]
 
+    # TODO what should we do if we build for both darwin and linux ?
+    # Should this only be affected by the host ?
     if platform == "linux":
```

Comment:
This should remain untouched according to me.

Reply:
It depends on the host running the build, not remote builders.